avm/regions.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I wonder if this PR should also undo the tempLocations introduced for some module tests or if we should do that in a dedicated PR

Reply:
I'd suggest to include the change in this PR

Reply:
Anyway, as per discussion, this json file should not be needed. We should retrieve the available locations for each resource via PS cmdlet. What we can keep is the logic to exclude specific locations known to have capacity issues. Please correct me if I'm missing something

Reply:
I thought the agreed approach is still to use the json. especially for where we need to do SKU level exceptions for certain regions then we have a centrally controlled file for the module owner.

Reply:
Not sure I got the same, let's sync offline on that

Reply:
I agree with @eriqua - BUT - think we agreed to keep the script to be able to still outright exclude locations. Together with the other location PR I think we should have a call sometime to finalize this.

Reply:
Lets get this booked in